### ISSUE TITLE
Ganglioglioma

### ISSUE THREAD

**Issue by DLewisSmith** [obophenotype/human-phenotype-ontology#6732]
**Preferred term label:**
Ganglioglioma
**Synonyms**

**Definition (free text, please give PubMed ID)**
Gangliogliomas are rare tumours of the central nervous system. The are WHO grade I and considered low grade. They are well differentiated neuroepithelial tumours consisting of both dysplastic neuronal and neoplastic glial cells.

Comment: They are commonly found in the temporal lobe and may be associated with temporal lobe epilepsy.

PMID: 25408925

**Parent term (use hpo.jax.org/app)**
Neuronal/glioneuronal neoplasm of the central nervous system HP:0025170

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Temporal lobe epilepsy

**Your nano-attribution (ORCID)**
0000-0002-1735-8178

**Comment by pnrobinson** [comment]
added
